src/styles/stylesheet.css

```diff
@@ -20,8 +27,10 @@ body {
 }
 
 .content {
-  background-color: white;
-  color: #404040;
+  margin: 0 auto;
+  width: 1140px;
+  height: 650px;
+  color: var(--secondary-color);
 }
```

Comment:
Change class name to `.container` and remove static `width` field. Look at the behavior page in different size.